main/lsp/tools/make_lsp_types.cc

```diff
@@ -56,6 +56,7 @@ shared_ptr<JSONType> makeArray(shared_ptr<JSONType> type) {
 
 void makeLSPTypes(vector<shared_ptr<JSONClassType>> &enumTypes, vector<shared_ptr<JSONObjectType>> &classTypes) {
     // Singletons
+    shared_ptr<JSONType> JSONAny = make_shared<JSONObjectType>();
```

Comment:
do you mind adding a comment here that mentions that the LSP spec actually allows more than just object types here? (it allows anything, including arrays, strings, numbers, and null)

Reply:
I haven't pushed that yet, but I've declared `JSONAny` as
```
    // In teh LSP specification JSONAny defined as
    // `export type LSPAny = LSPObject | LSPArray | string | integer | uinteger | decimal | boolean | null;`
    // https://microsoft.github.io/language-server-protocol/specifications/specification-3-17/#number
    //
    // We don't support LSPObject and LSPArray options yet
    shared_ptr<JSONType> JSONAny = makeVariant({JSONNull, JSONBool, JSONInt, JSONDouble, JSONString});
```